### PR TITLE
another step towards seperating asset service and market service func…

### PR DESCRIPTION
**Pull request by elmutt**
## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [x] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

## Description

Use data from asset service instead of functionality from market service

## Testing

Please outline all testing steps

1. Pull branch locally and run `yarn` to install new deps
2. etc...

### CODE REVIEW

**Review by SterL — APPROVED** [review]

**Review by DaoDev44 — APPROVED** [review]

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["react-app", "react-app/jest", "plugin:jsx-a11y/recommended", "prettier"],
-  "plugins": ["jest", "prettier", "simple-import-sort"],
```
Comment (bootsboogie) [inline .eslintrc:3]: Did we want to remove this?

Comment (elmutt) [reply .eslintrc:3]: why would anyone want this!?

Comment (elmutt) [reply .eslintrc:3]: oh shit maybe not the whole line

Comment (technojak) [reply .eslintrc:3]: Because organized code is nice

**Pull request merged**